web/integration_test.go

```diff
@@ -261,22 +261,24 @@ func TestIntegration_ExternalAdapter_RunLogInitiated(t *testing.T) {
 
 	logBlockNumber := 1
 	logs <- cltest.NewRunLog(j.ID, cltest.NewAddress(), logBlockNumber, `{}`)
-	time.Sleep(100 * time.Millisecond)
+	cltest.WaitForRuns(t, j, app.Store, 1)
 	jrs, err := app.Store.JobRunsFor(j.ID)
 	assert.Nil(t, err)
 	jr := jrs[0]
 	assert.True(t, jr.Status.PendingConfirmations())
 
-	newHeads <- models.BlockHeader{Number: cltest.BigHexInt(logBlockNumber + 1)}
-	time.Sleep(100 * time.Millisecond)
-	assert.True(t, jr.Status.PendingConfirmations())
+	g := gomega.NewGomegaWithT(t)
 	newHeads <- models.BlockHeader{Number: cltest.BigHexInt(logBlockNumber + 9)}
-	time.Sleep(100 * time.Millisecond)
-	assert.True(t, jr.Status.PendingConfirmations())
+	g.Eventually(func() bool {
+		jr = cltest.FindJobRun(app.Store, jr.ID)
+		return jr.Status.PendingConfirmations()
+	}).Should(gomega.BeTrue())
```

Comment:
Rather than rolling out `gomega` here, can we use our helpers `cltest.WaitForJobRunToPendConfirmations` and `cltest.WaitForJobRunToPendCompletions`?

https://github.com/smartcontractkit/chainlink/blob/master/internal/cltest/cltest.go#L426-L442